usage-examples/go/atlas-sdk-go/internal/auth/client.go

```diff
@@ -0,0 +1,25 @@
+package auth
+
+import (
+	"atlas-sdk-go/internal/config"
+	"context"
+	"fmt"
+
```

Comment:
Nit: I don't think this newline is needed, unless there's a style convention here I'm not familiar with?

Reply:
yeah, the newline is a [style convention](https://google.github.io/styleguide/go/decisions.html#import-grouping) - group imports by package source, and each grouped section is separated by a blank line. but actually, I'm glad you pointed this out because the local imports should be grouped at the bottom. I just ran `goimports` to apply it and update imports across project